main.py

```diff
@@ -68,13 +92,33 @@ def parserinit():
     mod = core.ModuleManager()
     mod.loadModule(cfg, args)
 
+    
     if args.list_mods:
         print('Available Modules (include system utilities)')
         print('====================================')
         for m in mod.modules:
             print(m)
         print('====================================')
-    elif args.server:
-        server.start(args, cfg, mod)
+    #elif args.server:
+    #    server.start(args, cfg, mod)
```

Comment:
Wait a second, are you completely removing the server functionality and making it a GUI-only application? I am afraid, that I cannot merge this PR here, if this is the case.